ensembl/conf/json/homo_sapiens_vcf.json

```diff
@@ -14,110 +14,119 @@
+      "id": "gnomADe_GRCh38",
+      "description": "Genome Aggregation Database exomes r2.1",
       "species": "homo_sapiens",
       "assembly": "GRCh38",
       "type": "local",
-      "filename_template": "/homo_sapiens/GRCh38/variation_genotype/gnomad/r2.1/genomes/gnomad.genomes.r2.1.sites.grch38.chr###CHR###_noVEP.vcf.gz",
+      "filename_template": "/homo_sapiens/GRCh38/variation_genotype/gnomad/r2.1/exomes/gnomad.exomes.r2.1.sites.grch38.chr###CHR###_noVEP.vcf.gz",
       "chromosomes": [
         "1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14",
-        "15", "16", "17", "18", "19", "20", "21", "22", "X"
+        "15", "16", "17", "18", "19", "20", "21", "22", "X", "Y"
       ],
-      "population_prefix": "gnomADg:",
+      "population_prefix": "gnomADe:",
       "population_display_group": {
-        "display_group_name": "gnomAD genomes",
-        "display_group_priority": 1.5
+        "display_group_name": "gnomAD exomes",
+        "display_group_priority": 1.4
       },
       "populations": {
-        "9900000": {
-          "name": "gnomADg:ALL",
+        "9900010": {
+          "name": "gnomADe:ALL",
           "_raw_name": "",
-          "description": "All gnomAD genomes individuals"
+          "description": "All gnomAD exomes individuals"
         },
-        "9900001": {
+        "9900011": {
           "name": "afr",
           "description": "African/African American"
         },
-        "9900002": {
+        "9900012": {
           "name": "amr",
           "description": "Latino"
         },
-        "9900003": {
+        "9900013": {
           "name": "asj",
           "description": "Ashkenazi Jewish"
         },
-        "9900004": {
+        "9900014": {
           "name": "eas",
           "description": "East Asian"
         },
-        "9900005": {
+        "9900015": {
           "name": "fin",
           "description": "Finnish"
         },
-        "9900006": {
+        "9900016": {
           "name": "nfe",
           "description": "Non-Finnish European"
         },
-        "9900007": {
+        "9900017": {
           "name": "oth",
           "description": "Other"
+        },
+        "9900018": {
+          "name": "sas",
+          "description": "South Asian"
         }
       }
     },
     {
-      "id": "gnomADe_GRCh38",
-      "description": "Genome Aggregation Database exomes r2.1",
+      "id": "gnomADg_r3.0_GRCh38",
+      "description": "Genome Aggregation Database genomes r3.0",
       "species": "homo_sapiens",
       "assembly": "GRCh38",
-      "type": "local",
-      "filename_template": "/homo_sapiens/GRCh38/variation_genotype/gnomad/r2.1/exomes/gnomad.exomes.r2.1.sites.grch38.chr###CHR###_noVEP.vcf.gz",
+      "type": "remote",
+      "filename_template": "ftp://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh38/variation_genotype/gnomad/r3.0/genomes/gnomad.genomes.r3.0.sites.chr###CHR###_trimmed_info.vcf.bgz",
```

Comment:
Hi @at7 ;
just making sure the change to use the ftp url for this file is intended because the others haven't changed.